src/wrapper_generators.jl

```diff
@@ -2,6 +2,22 @@ include("products/executable_generators.jl")
 include("products/file_generators.jl")
 include("products/library_generators.jl")
 
+macro generate_wrapper_header(src_name)
```

Comment:
Does the JLLs need to be updated to call this?

Reply:
Yes, this will need to be added to BB.

Reply:
https://github.com/JuliaPackaging/BinaryBuilder.jl/commit/7314f5f248767f21c18d0a85d8412539e5a5c6d0